src/orion/core/worker/trial.py

```diff
@@ -250,7 +247,11 @@ def hash_name(self):
         """
         if not self.params:
```

Comment:
Should add self.experiment in the checks here.

Reply:
What's the second change?

Reply:
`if not self.params or not self.experiment`